### PR TITLE
SensorOverlay

### PR DESCRIPTION
**Pull request by xitman78**
Basic SensorOverlay Component.
This component depends on npm module "react-odometerjs" which requires CSS theme usually loaded from CDN. To avoid this CSS dependency I created styled-component
 wrapper "StyledOdometer".

Need to add unit tests later.

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-74.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-74.gearboxjs.preview.cogniteapp.com

File: src/components/SensorOverlay/SensorOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  Timeseries,
+  SensorOverlayClickHandler,
+  LinkType,
+} from 'utils/validators/SensorOverlayTypes';
+
+export const defaultStrings: VMetadata = {
+  unknownValue: 'unknownValue',
+};
+
+const SensorOverlayContainer = styled.div`
```
Comment (Kamillowl) [inline src/components/SensorOverlay/SensorOverlay.tsx]: Are there enough styled components that they should maybe be in their own file?

File: src/components/SensorOverlay/SensorOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    top: number;
+  };
+  distToCamera: number;
+}
+
+export interface LabelPositionMap {
+  [labelName: string]: LabelPosition;
+}
+
+export type IdType = number | string;
```
Comment (Kamillowl) [inline src/components/SensorOverlay/SensorOverlay.tsx]: we already have a type like this in validators (VId ). Better use that one

**Comment by xitman78** [comment]
@Kamillowl @f1cognite I'm closing this PR since the component will be totally remade based on DraggableBox and DraggablePoint, etc.

**Pull request closed without merging**